test/controllers/update/overseas.entity.query.controller.spec.ts

```diff
@@ -55,7 +55,7 @@ describe("OVERSEAS ENTITY QUERY controller", () => {
     });
 
     test(`renders the ${config.OVERSEAS_ENTITY_QUERY_PAGE} page with value if exists`, async () => {
-      mockGetApplicationData.mockReturnValueOnce({ oe_number: 'OE123456' });
+      mockGetApplicationData.mockReturnValueOnce({ oe_number: '12345678' });
```

Comment:
The oe_number will need to be OE123456 as the validation check has been merged to main, and this test will fail

Reply:
Tests passed with current value but updated to use OE123456